tgui/packages/tgui/interfaces/CommunicationsConsole.js

```diff
@@ -0,0 +1,667 @@
+            />
+        }
+
+        {!!shuttleCalledPreviously && (
+          shuttleLastCalled && (
+            <Box>
+              Most recent shuttle call/recall traced to:
+              {" "}<b>{shuttleLastCalled}</b>
+            </Box>
+          ) || (
+            <Box>Unable to trace most recent shuttle/recall signal.</Box>
+          )
+        )}
+      </Section>
+
+      {!!canSetAlertLevel && (
+        <Section title="Alert Level">
+          <Flex justify="space-between">
+            <Flex.Item>
+              <Box>
+                Currently on <b>{capitalize(alertLevel)}</b> Alert
+              </Box>
+            </Flex.Item>
+
+            <Flex.Item>
+              <AlertButton
+                alertLevel="green"
+                showAlertLevelConfirm={showAlertLevelConfirm}
+                setShowAlertLevelConfirm={setShowAlertLevelConfirm}
+              />
+
+              <AlertButton
+                alertLevel="blue"
+                showAlertLevelConfirm={showAlertLevelConfirm}
+                setShowAlertLevelConfirm={setShowAlertLevelConfirm}
+              />
+            </Flex.Item>
+          </Flex>
+        </Section>
+      )}
+
+      <Section title="Functions">
+        <Flex
+          direction="column">
+          {!!canMakeAnnouncement && <Button
+            icon="bullhorn"
+            content="Make Priority Announcement"
+            onClick={() => act("makePriorityAnnouncement")}
+          />}
+
+          {!!canToggleEmergencyAccess && <Button.Confirm
+            icon="id-card-o"
+            content={`${emergencyAccess ? "Disable" : "Enable"} Emergency Maintenance Access`}
+            color={emergencyAccess ? "bad" : undefined}
+            onClick={() => act("toggleEmergencyAccess")}
+          />}
+
+          <Button
+            icon="desktop"
+            content="Set Status Display"
+            onClick={() => act("setState", { state: STATE_CHANGING_STATUS })}
+          />
+
+          <Button
+            icon="envelope-o"
+            content="Message List"
+            onClick={() => act("setState", { state: STATE_MESSAGES })}
+          />
+
+          {(canBuyShuttles !== 0) && <Button
+            icon="shopping-cart"
+            content="Purchase Shuttle"
+            disabled={canBuyShuttles !== 1}
+            // canBuyShuttles is a string detailing the fail reason
+            // if one can be given
+            tooltip={canBuyShuttles !== 1 ? canBuyShuttles : undefined}
+            tooltipPosition="right"
+            onClick={() => act("setState", { state: STATE_BUYING_SHUTTLE })}
+          />}
+
+          {!!canMessageAssociates && <Button
+            icon="comment-o"
+            content={`Send message to ${emagged ? "[UNKNOWN]" : "CentCom"}`}
+            disabled={!importantActionReady}
+            onClick={() => setMessagingAssociates(true)}
+          />}
+
+          {!!canRequestNuke && <Button
+            icon="bomb"
```

Comment:
```suggestion
            icon="radiation"
```
most important pr review of 2020

Reply:
Got overlooked :sadcat:

Reply:
I can do it today.